src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostConfirm/ActualCostConfirm.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import {
+  Button,
+  Col,
+  KeyValue,
+  Modal,
+  ModalFooter,
+  Row,
+} from '@folio/stripes/components';
+
+import { getPatronName } from '../../util';
+
+import {
+  ACTUAL_COST_CONFIRM_MODAL_DEFAULT,
+  ACTUAL_COST_DEFAULT,
+  ACTUAL_COST_RECORD_FIELD_NAME,
+  ACTUAL_COST_RECORD_FIELD_PATH,
+  ACTUAL_COST_TYPES,
+  ACTUAL_COST_PROP_TYPES,
+  DEFAULT_VALUE,
+} from '../../../../../constants';
+
+import css from './ActualCostConfirm.css';
+
+const ActualCostConfirm = ({
```

Comment:
I would rename it to `ActualCostConfirmModal`. What do you think about it?
In my opinion, the name should say that it is a modal window.

Reply:
Fixed